R/diseases.R

```diff
@@ -12,12 +12,9 @@ montagu_diseases <- function(location = NULL) {
     name = vcapply(res, "[[", "name"))
 }
 
-##' Diseases have an internal id and a name in Montagu, which are generally,
-##' but not always the same. Occasionally, it is necessary to provide the
-##' disease id to Montagu functions.
 ##' @title Retrieve disease with given id.
 ##' @param disease_id The disease id.
-##' @inheritParams montagu_diseases
+##' @inherit montagu_diseases
```

Comment:
What is the motivation here? Looks like we're still only dragging in parameters?

Reply:
The intention (rightly or wrongly!) is that the bit of text at the top (Diseases have an internal id and a name in Montagu, etc....) - gets printed on both ?montagu_disease and ?montagu_diseases - which come up on different pages at the moment. Probably they should be on the same page anyway.